test/parallel/test-tls-alert.js

```diff
@@ -32,15 +31,17 @@ if (!common.hasCrypto) {
   common.skip('missing crypto');
   return;
 }
-const tls = require('tls');
 
+const assert = require('assert');
+const { spawn } = require('child_process');
```

Comment:
@sam-github? Ref: https://github.com/nodejs/node/commit/02b92704448bc2784bd436aea3f97e76f3f0ec16

Since the whole paragraph's subject is modules, my intuition says sort by module name.
<sub>(also congruent with Jewish law that states you bless food according to the main ingredient and not by how you eat it 😉)</sub>